features/volledigeNaam.feature

```diff
@@ -114,38 +102,38 @@ Rule: een adellijke titel wordt opgenomen tussen de voornamen en het voorvoegsel
 
     Voorbeelden:
```

Comment:
Moeten hier niet ook voorbeelden opgenomen worden waarbij een of meer kolommen een standaardwaarde hebben?